### DISCUSSION TITLE
Advanced object search 4.1 not connect to elastic search

### DISCUSSION THREAD

**Discussion by Rojal123245** [pimcore/pimcore#10023] [Q&A / Support]
I am having difficulty to connect my elastic search server in pimcore 6.9.2 . ongr/elasticsearch-dsl:7.2.0 elasticsearch/elasticsearch :7.14 .phpversion=7.2
my advanced object search configuration has been added in app/config/config.yml
advanced_object_search:
    # Prefix for index names
    index_name_prefix: 'advanced_object_search_'   
    
    # List of elasticsearch hosts
    es_hosts:
        - 'elasticsearchip:9200'

**Comment by ghost** [comment]
> I am having difficulty to connect my elastic search server in pimcore 6.9.2 . ongr/elasticsearch-dsl:7.2.0 elasticsearch/elasticsearch :7.14 .phpversion=7.2
> my advanced object search configuration has been added in app/config/config.yml
> advanced_object_search:
> # Prefix for index names
> index_name_prefix: 'advanced_object_search_'
> 
> ```
> # List of elasticsearch hosts
> es_hosts:
>     - 'elasticsearchip:9200'
> ```

Hi @Rojal123245

Maybe your elasticsearch version is creating the issue, I was facing some issues with elasticsearch and I found the issue with the ES version (Like the Pimcore is not support elsticsearch 7).

Please check the below Link maybe it will help you.

https://github.com/pimcore/advanced-object-search/issues/108

https://githubmemory.com/repo/pimcore/advanced-object-search/issues?cursor=Y3Vyc29yOnYyOpK5MjAyMS0wMS0xOFQyMzo0NjoyNSswODowMM4u_Zyq&pagination=next&page=2

Also, you can use only IP without Port
es_client_params:
                        hosts:
                            - '127.0.0.1'

**Reply by Rojal123245** [reply]
hi @rushit-credencys  
thank you for the reply my es version is 7.14 should i downgrade to es 6 ?

**Reply by ghost** [reply]
Yes @Rojal123245, You need to downgrade.

**Reply by Rojal123245** [reply]
@rushit-credencys  after downgrade i am still getting same problem can it be other dependent composer pakage like ongr elasticsearch-dsl or any other
